hoomd/hpmc/update.py

```diff
@@ -15,387 +15,138 @@
-        return {'delta': self.length_delta, 'weight': self.length_weight};
+        length (dict):
+            Enable/disable isobaric box length moves and set parameters (change box lengths independently).
+            The dictionary has the following keys:
 
-    def shear(self,  delta=None, weight=None, reduce=None):
-        R""" Enable/disable box shear moves and set parameters.
+            * ``weight`` (float) - maximum change of the box thickness for each pair of parallel planes
+                    connected by the corresponding box edges. I.e. maximum change of HOOMD-blue box parameters Lx, Ly, Lz.
+            * ``delta`` (list or tuple) -  maximum change of the box lengths Lx, Ly, Lz.
 
-        Args:
-            delta (tuple): maximum change of the box tilt factor xy, xz, yz.
-            reduce (float): Maximum number of lattice vectors of shear to allow before applying lattice reduction.
-                    Shear of +/- 0.5 cannot be lattice reduced, so set to a value < 0.5 to disable (default 0)
-                    Note that due to precision errors, lattice reduction may introduce small overlaps which can be
-                    resolved, but which temporarily break detailed balance.
-            weight (float): relative weight of this box move type relative to other box move types. 0 disables this
-                            move type.
+        shear (dict):
+            Enable/disable isobaric box shear moves and set parameters. The dictionary
+            has the following keys:
 
-        Sample the distribution of box shear by adjusting the HOOMD-blue tilt factor parameters xy, xz, and yz.
+            * ``weight`` (float) - relative weight of shear box moves relative to other box move types.
+            * ``delta`` (list or tuple) -  maximum change of the box tilt factor xy, xz, yz.
+            * ``reduce`` (float) - Maximum number of lattice vectors of shear to allow before applying lattice reduction.
+                     Shear of +/- 0.5 cannot be lattice reduced, so set to a value < 0.5 to disable (default 0)
+                     Note that due to precision errors, lattice reduction may introduce small overlaps which can be
+                     resolved, but which temporarily break detailed balance.
 
-        Note:
-            When an argument is None, the value is left unchanged from its current state.
+    Note:
 
-        Example::
+        One or more Monte Carlo move types may be applied to evolve the simulation box. By default,
+        no moves are applied (all *delta* and *weight* values for all move types default to 0).
 
-            box_update.shear(delta=(0.01, 0.00, 0.0)) # 2D box changes
-            box_update.shear(delta=(0.01, 0.01, 0.01), weight=2)
-            box_update.shear(delta=(0.10, 0.01, 0.01), weight=0.15) # sample xy more aggressively
+        Pressure inputs to update.BoxMC are defined as :math:`\beta P`. Conversions from a specific definition of reduced
```

Comment:
Pressure units are described in the tutorial and do not need to be mentioned here.